website/pages/en/quick-start.mdx

```diff
@@ -2,24 +2,20 @@
 title: Quick Start
 ---
 
-This guide will quickly take you through how to initialize, create, and deploy your subgraph to Subgraph Studio.
+This guide walks you through the basics of creating, deploying, publishing, and querying a subgraph on The Graph easily.
```

Comment:
Too many terms but technically publishing already covers - creating & deploying so we can just say this

```suggestion
This guide walks you through the basics of publishing and querying a subgraph on The Graph easily.
```